dotcom-rendering/src/web/components/Callout/MessageUs.tsx

```diff
@@ -138,7 +138,7 @@ const MessageUs: FC<{ contacts: CalloutContactType[] }> = ({ contacts }) => {
 						priority="primary"
 						href={`${contact.urlPrefix}${contact.value}`}
 						target="_blank"
-						rel="noopener"
+						rel="noreferrer noopener"
```

Comment:
We should just be able to specify `rel="noreferrer"`

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer

Reply:
thanks @joecowton1! I'd misread a comment about browser support which I took to imply that there were some edge cases where `noopener` was supported but `noreferrer` wasn't, but on re-reading it looks like it's the other way around!